tests/projects/test_serializers.py

```diff
@@ -0,0 +1,123 @@
+
+    # active phase
+    phase_factory(
+        start_date=last_week,
+        end_date=next_week,
+        module__project=project_active,
+    )
+
+    # future phase
+    phase_factory(
+        start_date=tomorrow,
+        end_date=next_week,
+        module__project=project_future,
+    )
+
+    # active phase
+    phase_factory(
+        start_date=yesterday,
+        end_date=tomorrow,
+        module__project=project_active_and_future,
+    )
+
+    # future_phase
+    phase_factory(
+        start_date=tomorrow,
+        end_date=next_week,
+        module__project=project_active_and_future,
+    )
+
+    # past phase
+    phase_factory(
+        start_date=last_week,
+        end_date=yesterday,
+        module__project=project_past,
+    )
+
+    with freeze_time(now):
+        projects = Project.objects.all().order_by("created")
+
+        project_serializer = ModerationProjectSerializer(projects, many=True, now=now)
+        project_data = project_serializer.data
+        assert len(project_data) == 4
+
+        assert project_data[0]["title"] == "active"
+        assert project_data[1]["title"] == "future"
+        assert project_data[2]["title"] == "active and future"
+        assert project_data[3]["title"] == "past"
+
+        assert project_data[0]["participation_string"] == _("running")
+
+        assert project_data[2]["participation_string"] == _("running")
+        assert project_data[3]["participation_string"] == _("completed")
+
+        assert project_data[0]["active_phase"][0] == 50
+        assert "7" in project_data[0]["active_phase"][1]
+        assert not project_data[1]["active_phase"]
+        assert project_data[2]["active_phase"][0] == 50
+        assert "1" in project_data[2]["active_phase"][1]
+        assert not project_data[3]["active_phase"]
+
+        assert not project_data[0]["future_phase"]
+        assert not project_data[3]["future_phase"]
+
+        assert not project_data[0]["past_phase"]
+        assert not project_data[1]["past_phase"]
+        assert not project_data[2]["past_phase"]
+
+        assert project_data[0]["participation_active"]
+        assert project_data[1]["participation_active"]
+        assert project_data[2]["participation_active"]
+        assert not project_data[3]["participation_active"]
+
+        assert project_data[0]["tile_image"]
+        assert not project_data[1]["tile_image"]
+        assert not project_data[2]["tile_image"]
+        assert project_data[3]["tile_image"]
+
+        assert project_data[0]["tile_image_copyright"]
+        assert not project_data[1]["tile_image_copyright"]
+        assert not project_data[2]["tile_image_copyright"]
+        assert project_data[3]["tile_image_copyright"]
+
+        assert not project_data[0]["tile_image_alt_text"]
+        assert not project_data[1]["tile_image_alt_text"]
+        assert not project_data[2]["tile_image_alt_text"]
+        assert project_data[3]["tile_image_alt_text"]
```

Comment:
I just removed all the date format tests as we don't use the same as mB and thought wouldn't be missed as I would actually just like to test the copyright and alt